packages/sanity/src/desk/panes/document/timeline/timelineMenu.tsx

```diff
@@ -161,7 +155,7 @@ export function TimelineMenu({chunk, mode, placement}: TimelineMenuProps) {
       ref={setPopover}
     >
       <Button
-        disabled={!ready || isDeleted}
+        disabled={!ready}
```

Comment:
**suggestion**: we don't disable timeline menu items _for now_, as doing so means you're unable to navigate review changes on deleted documents

there's probably a greater question over whether review changes should even be visible in deleted documents to begin with – this whole area is probably due for a bit of a visual overhaul

Reply:
Agreed, we can tackle this later. Seems to work well regardless.